daisy_workflows/build-publish/rhui/README.md

```diff
@@ -0,0 +1,123 @@
+# RHUI node image builds
+
+The Red Hat Update Infrastructure is a package mirror service with client TLS
+authentication, offered by Red Hat and operated on Cloud Providers. This
+directory contains the build workflows and artifacts for creating deployable
+node images for RHUI's RHUA and CDS nodes. We do not use the HAProxy node type,
+instead offering RHUI behind a Google Cloud Internet Load Balancer.
+
+We are currently building RHUA and CDS nodes using the v4.2 installation media.
```

Comment:
nit: Would remove the version # to avoid having to update the contents of the README each time. Could just point to the install / config location that references the version (or just say we are installing using the installation media).